website/pages/en/deploying/deploying-a-subgraph-to-studio.mdx

```diff
@@ -25,44 +44,103 @@ yarn global add @graphprotocol/graph-cli
+Then, use the following command to authenticate from the CLI:
 
 ```bash
 graph auth --studio <DEPLOY KEY>
 ```
 
-## Deploying a Subgraph to Subgraph Studio
+## Deploying a Subgraph
 
-Once you are ready, you can deploy your subgraph to Subgraph Studio. Doing this won't publish your subgraph to the decentralized network, it will only deploy it to your Studio account where you will be able to test it and update the metadata.
+Once you are ready, you can deploy your subgraph to Subgraph Studio.
 
-Here is the CLI command that you need to use to deploy your subgraph.
+> Deploying a subgraph with the CLI pushes it to the Studio, where you can test it and and update the metadata. This action won't publish your subgraph to the decentralized network.
+
+Use the following CLI command to deploy your subgraph:
 
 ```bash
 graph deploy --studio <SUBGRAPH_SLUG>
 ```
 
-After running this command, the CLI will ask for a version label, you can name it however you want, you can use labels such as `0.1` and `0.2` or use letters as well such as `uniswap-v2-0.1`. Those labels will be visible in Graph Explorer and can be used by curators to decide if they want to signal on this version or not, so choose them wisely.
+After running this command, the CLI will ask for a version label.
+
+- It's strongly recommended to use [semver](https://semver.org/) for versioning like `0.0.1`. That said, you are free to choose any string as version such as:`v1`, `version1`, `asdf`.
+- The labels you create will be visible in Graph Explorer and can be used by curators to decide if they want to signal on a specific version or not, so choose them wisely.
+
+## Testing your Subgraph
+
+After deploying, you can test/stage your subgraph in Subgraph Studio, deploy another version, update the metadata, and publish to [Graph Explorer](https://thegraph.com/explorer) when you are ready.
+
+Use Subgraph Studio’s playground to check your subgraph logs and see where a subgraph fails.
+
+## Publish your Subgraph
+
+In order to publish your subgraph successfully, review [publishing a subgraph](/publishing/publishing-a-subgraph/).
+
+## Versioning your Subgraph with the CLI
+
+If you want to update your subgraph, you can do the following:
+
+- You can deploy a new version to the Studio using the CLI (it will only be private at this point).
+- Once you're happy with it, you can publish your new deployment to [Graph Explorer](https://thegraph.com/explorer).
+- This action will create a new version of your subgraph that curators can start signaling on and Indexers can index.
+
+You can also update your subgraphs' metadata without publishing a new version. You can update your subgraph details in the Studio (under the profile picture, name, description, etc.) by checking an option called **Update Details** in Graph Explorer. If this is checked, an on-chain transaction will be generated that updates subgraph details in the Explorer without having to publish a new version with a new deployment.
```

Comment:
```suggestion
You can also update your subgraph's metadata without publishing a new version. You can update your subgraph details in Studio (under the profile picture, name, description, etc.) by checking an option called **Update Details** in Graph Explorer. If this is checked, an on-chain transaction will be generated that updates subgraph details in Explorer without having to publish a new version with a new deployment.
```

Also, I find this whole paragraph a bit confusing. Is "Update Details" a checkbox that exists in Studio? I think a screenshot would help greatly here.